### ISSUE TITLE
?meetup follow-me  potential cross-message edits

### ISSUE THREAD

**Issue by Aprillion** [kentcdodds/kcd-discord-bot-v1#50]
Just watching live stream where you created meetups and line 49 in https://github.com/kentcdodds/kcd-discord-bot/blob/7686deebc6aff3155bca8c02ef116ce73fbf20d6/src/commands/command-fns/meetup.js#L49 did not seem to be designed for the following, so here is a potential scenario for `msg.content.includes(message.author.id)` matching the first message where the author is mentioned:

* given person A mentions person B in their brief description like `?meetup follow-me doing a collab with @B`,
* when person B wants to make a new message with `?meetup follow-me I will also stream my own thing`,
* the command will (try to) edit the existing message from person A

..it could be even more interesting if `followMeChannel.messages.cache` iterates from newest to oldest 😂

Let me know if you would worry about this scenario at all. (I can try to investigate more later during the week, if not closed by then)

**Comment by Aprillion** [comment]
The line also got copied into https://github.com/kentcdodds/kcd-discord-bot/blob/7686deebc6aff3155bca8c02ef116ce73fbf20d6/src/meetup/utils.js#L24

**Comment by kentcdodds** [comment]
Yeah, we currently act under the assumption that only one user will be mentioned in these messages and it will be the host, but that's not always a great assumption. We should probably regex it out.
